pom.xml

```diff
@@ -85,8 +85,9 @@
         <jackson.annotations.version>${jackson.version}</jackson.annotations.version>
 
         <!--JAKARTA-->
-        <com.sun.jakarta.mail.version>2.0.1</com.sun.jakarta.mail.version>
         <jakarta.persistence.version>3.1.0</jakarta.persistence.version>
+        <jakarta.mail.version>2.1.1</jakarta.mail.version>
+        <angus.mail.version>2.0.1</angus.mail.version>
```

Comment:
Do we still need `angus.mail.version` property?

Reply:
Current state: Yes, we do, as we need to provide an implementation for the Jakarta Mail API.